frontend/webapp/static/js/custom/graphql.js

```diff
@@ -235,3 +234,30 @@ function fetchConfigStatsLive(ws, cb_func) {
         configStatsCalled = true;
     }
 }
+
+function fetchDBVersion() {
```

Comment:
can you try this: https://stackoverflow.com/questions/45399923/eslint-disable-warning-defined-but-never-used-for-specific-function

to check if it works and then fix the other eslint errors

Reply:
outdated with new fixes